examples/countTransferManager.ts

```diff
@@ -20,16 +19,14 @@ window.addEventListener('load', async () => {
 
   // Instantiate the API
   const polymathAPI = new PolymathAPI(params);
-  // Get some poly tokens in your account and the security token
-  const web3Wrapper = new Web3Wrapper(params.provider, {
-    gasPrice: params.defaultGasPrice,
-  });
-  const address = await web3Wrapper.getAvailableAddressesAsync();
+  const myAddress = await polymathAPI.getAccount();
 
-  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
-  await polymathAPI.getPolyTokens({
-    amount: new BigNumber(1000000),
-    address: await polymathAPI.securityTokenRegistry.address(),
+  // Get some poly tokens in your account and the security token
+  // Poly token faucet works on test net
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000) });
+  await polymathAPI.polyToken.transfer({
```

Comment:
Sorry, my bad. You don't need to transfer. You only need to approve as you are doing below, then the ST calls `transferFrom`. So this entire line is not needed.

Reply:
Okay I tested it and saw that, I will go ahead and just remove that transfer from my files